### PR TITLE
[64] Update README with new Release Management

### PR DESCRIPTION
**Pull request by irushavm**
## Description

This change updates the release management and branching strategies used for this project

## Related Issue
#64 

## Motivation and Context
This change is proposed as an alternative to the current approach which switches the current development branch based on the most recent release cut.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -72,34 +72,71 @@ The MEMS project is based on the Reveal Foundation to provide the scaffold to bu
...
+
+Bugfix branch format: `bugfix/<ISSUE_ID>/<description>`
+
+For example:
+```bash
+git checkout -b bugfix/22/request-id-unknown-in-respon
+```
+
+### Release Branches
+Release branches are long lived-branches that reflect the state of the product at ehe end of a milestone.
```
Comment (akshaypuli) [inline README.md:119]: Minor spelling mistake. "ehe" -> "the"

**Review by akshaypuli — DISMISSED** [review]

**Comment by akshaypuli** [comment]
I am in favor of the new strategy! Thanks @irushavm for putting this together!

**Review by oaedward — APPROVED** [review]
Great

File: README.md [hunk 2]
```diff
@@ -72,34 +72,71 @@ The MEMS project is based on the Reveal Foundation to provide the scaffold to bu
...
-### Branching Strategy
+## Release Management
 
-MAIN -> RC04
-- Change main application version number
-- Version: 0.4.0 <Release>.<Release Candidate>.<Minor Changes>
-  
-RC04 -> Feature Branch (RC04_NewViewVendor)
-  - Single contributor
+### Verioning
```
Comment (oaedward) [inline README.md:77]: Missed an "s" in Versioning

**Review by oaedward — DISMISSED** [review]

**Review by akshaypuli — APPROVED** [review]
Looks Great!!

**Review by oaedward — APPROVED** [review]

**Pull request merged**